src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs

```diff
@@ -1845,7 +1859,10 @@ public struct DeeplyNested
 
         public class BaseClassWithVirtualProperty
         {
+            private string? PrivateProperty { get; set; }
             public virtual string[] Test { get; set; } = System.Array.Empty<string>();
+
+            public string? ExposePrivatePropertyValue() => PrivateProperty;
```

Comment:
We should also add tests with different get/set combinations. For example:

```
class A 
{
   public virtual string TestString { get; set; }
}

class B : A
{
   public override string TestString
   {
      get { return ...; }
   }
}
```

Reply:
Fixed, thanks.